sklearn/metrics/tests/test_common.py

```diff
@@ -300,8 +292,6 @@
     "jaccard_similarity_score", "unnormalized_jaccard_similarity_score",
     "zero_one_loss", "unnormalized_zero_one_loss",
 
-    "precision_score", "recall_score", "f1_score", "f2_score", "f0.5_score",
```

Comment:
The `precision_score` is initiated with the default `'binary'` averaging. Previously this was converted to `'weighted'` for non-binary targets. Now that we are removing this behavior, this metric is not compatible with multilabel data.

So I'm removing it from this list to prevent it from getting tested with multilabel data.